client/src/i18n/en/form.json

```diff
@@ -503,6 +503,7 @@
             "ORDER_BY": "Order By",
             "PARENT_ACCOUNT": "Parent Account",
             "PASSIVE": "Passive",
+            "PAVION":"Pavion",
```

Comment:
In English, this would be `Pavillion`.  You can find the English translation in https://github.com/IMA-WorldHealth/bhima-2.X/issues/3408.